DataFlowDiagrams/DesigningForSSE.md

```diff
@@ -9,3 +9,57 @@
+
+## Threats Migitated
+
+|STRIDE|Count|
+|------|-----|
+|Spoofing|2|
+|Tampering|4|
+|Repudiation|10|
+|Information Disclosure|3|
+|Denial of Service|4|
+|Elevation of Privilege|3|
+
+
+## Needs Investigation
+
+|STRIDE|Count|
+|------|-----|
+|Spoofing|3|
+|Tampering|7|
+|Repudiation|1|
+|Information Disclosure|1|
+|Denial of Service|5|
+|Elevation of Privilege|3|
+
+Spoofing could occur in the “Bank Employee” external interactor if a malicious user pretends to be a verified user in order to get sensitive account information.  Tampering with any bits in the system could exist.  For example, if a dataflow contains JSON, JSON processing and hijacking threats may be exploited. Repudiation threats could result from letting everyone write to the logs.  Information disclosure threats appear when looking at who has access to the data in the system and if any of those with access are unauthorized.  Denial of Service threats are related to resource consumption.  Attacks could deadlock the system.  Elevation of Privilege threats could occur if an attacker passes data into the Elasticsearch Replica Node in order to change the flow of program execution.
+
+From our analysis of the automatically generated threats, it seems our biggest gaps in threat mitigation exist in tampering and denial of service.  Tampering threats exist in the API request from the bank employee to the Elasticsearch primary node, the index request from the Elasticsearch primary node to the file system, the index response from the file system to the Elasticsearch primary node, the replica index request from the Elasticsearch replica node to the replica file system, the replica index response from the replica file system to the Elasticsearch replica node, the replication request from the Elasticsearch primary node to the Elasticsearch replica node, and in the replication response from the Elasticsearch replica node to the Elasticsearch primary node.  Since this exists in 7 out of 8 data flows, the design seems particularly vulnerable to tampering threats.  The denial of service threats exist in the API request from the bank employee to the Elasticsearch primary node, exist in the API response from the Elasticsearch primary node to the bank employee, the index request from the Elasticsearch primary node to the file system, the replica index request from the Elasticsearch replica node to the replica file system, and the replication request from the Elasticsearch primary node to the Elasticsearch replica node.  The denial of service threat exists in 5 out of 8 data flows.  As a result, it is reasonable to conclude the design is also particularly vulnerable to denial of service threats.
```

Comment:
I don't think we need to list out all the places a threat exists, the professor can see that in the HTML report. I would summarize like you do when you say 'this exists in 7 out of 8 data flows...'